### DISCUSSION TITLE
pinDigestDisable preset is ignored for image in github action workflow

### DISCUSSION THREAD

**Discussion by lmilbaum** [renovatebot/renovate#26015] [Request Help]
### How are you running Renovate?

Self-hosted Renovate

### If you're self-hosting Renovate, select which platform you are using.

github.com

### Was this something which used to work for you, and then stopped?

I have not seen this working

### Describe the problem

Global renovate configurtion:
```
{
  "$schema": "https://docs.renovatebot.com/renovate-schema.json",
  "extends": [
    "github>platform-engineering-org/.github",
    ":pinDigestsDisabled"
  ],
  "automerge": false,
  "pinDigests": false
}
```

### Relevant debug logs

<details><summary>Logs</summary>

```


               {
                 "depName": "quay.io/centos-bootc/builder",
                 "currentValue": "latest",
                 "replaceString": "quay.io/centos-bootc/builder:latest",
                 "autoReplaceStringTemplate": "{{depName}}{{#if newValue}}:{{newValue}}{{/if}}{{#if newDigest}}@{{newDigest}}{{/if}}",
                 "datasource": "docker",
                 "depType": "container",
                 "updates": [
                   {
                     "isPinDigest": true,
                     "updateType": "pinDigest",
                     "newValue": "latest",
                     "newDigest": "sha256:8adb0f0b3db44d1fd94965c7e108db20b46afc78050faafe66626188fa7370c2",
                     "branchName": "renovate/all"
                   }
                 ],
```

</details>

### Have you created a minimal reproduction repository?

https://github.com/CentOS/centos-bootc/

**Comment by rarkins** [comment]
Please describe the problem and how to observe it. I don't see any evidence of a problem in https://github.com/CentOS/centos-bootc/issues/1

**Reply by lmilbaum** [reply]
I figured it out. The preset doesn't impact an image that is listed in a github actions workflow.
